### PR TITLE
Avoid warnings in multi_array_sort()

### PR DESCRIPTION
**Pull request by mariusbarth**
Hi @bgreiner,

I am not 100% sure, but I think I just fixed a bug in `tagsets/experiments.php`. From my understanding of `multi_array_sort()`, if values should be sorted by multiple other variables, these should be provided as an array, not a single character (that would have to explode()d in `multi_array_sort()`). I hope I'm not completely off here.

This should address issue "Undefined array key "time,experiment_name" in the bug tracker.

Warm regards and many thanks for maintaining this project!
Marius

### CODE REVIEW

File: tagsets/experiments.php [hunk 1]
```diff
@@ -336,7 +336,7 @@ function experiment__experiments_format_alist($alist) {
         <TD class="small" align="center">';
             if ($experiment_type=="laboratory") {
                 echo lang('sessions').': <span style="font-weight: bold;" title="number of sessions">'.$num_sessions.'</span>';
-                echo '&nbsp&nbsp&nbsp<span title="number of subjects assigned / enroled / participated"><i class="fa fa-users" style="font-size: 7pt; padding-right: 2px;"></i>'.
+                echo '&nbsp&nbsp&nbsp<span title="number of subjects assigned / enrolled / participated"><i class="fa fa-users" style="font-size: 7pt; padding-right: 2px;"></i>'.
```
Comment (bgreiner) [inline tagsets/experiments.php]: Please remove this from the commit. It has nothing to do with the issue, and it's a US vs. UK English thing.

File: tagsets/experiments.php [hunk 2]
```diff
@@ -782,7 +782,7 @@ function experiment__preload_experiments() {
         $result=or_query($query);
         while ($line=pdo_fetch_assoc($result)) { $experiments[$line['experiment_id']]['participated']='y'; }
 
-        $sort_order="time,experiment_name";
+        $sort_order=["time", "experiment_name"];
         multi_array_sort($experiments,$sort_order);
```
Comment (bgreiner) [inline tagsets/experiments.php:786]: Good catch. But please use traditional syntax here to keep backward compatibility. I.e., 
$sort_order=array("time","experiment_name");

**Comment by bgreiner** [comment]
Hi Marius,

Thanks, good catch. I added two comments to the commits. Happy to merge to develop then.

File: tagsets/experiments.php (hunk 2 above)
Comment (mariusbarth) [reply tagsets/experiments.php:786]: Done. :)

**Comment by mariusbarth** [comment]
Thanks, I just changed both issues accordingly. Sorry for the "typo correction" -- I made that change before I decided that I should try to get these tiny bug fixes upstream.

**Pull request merged**